source/deprecations/v1.x.html.md

```diff
@@ -1069,3 +1069,21 @@ c.swapNames();  // returns c
 Object.freeze(c);
 c.swapNames();  // EXCEPTION
 ```
+
+#### Ember CreateWithMixins
+
+`Ember.Object.createWithMixins` method has been deprecated. Instead call `Ember.Object.create` or `Ember.Object.extend`.
+
+```javascript
+var obj = Ember.Object.createWithMixins({
+
+});
+```
+
+Replace with code above with:
+
+```javascript
+var obj = Ember.Object.create({
+
+});
```

Comment:
I'd actually suggest:

```
var obj = Ember.Object.extend({
}).create();
```

As that is actually exactly what `createWithMixins` did.